test/test-pages/001/expected-metadata.json

```diff
@@ -0,0 +1,5 @@
+{
+  "title": "Get your Frontend JavaScript Code Covered | Code",
+  "byline": "Nicolas Perriault —",
+  "excerpt": "width=device-width"
```

Comment:
Feels odd to mark this kind of erroneous result as "expected". I get that we want a starting point, though I hope we won't forget to fix these ;)